From: KevinDuan <duankaiwen@kylinos.cn>
Date: Wed, 14 Aug 2024 03:20:34 +0000
Subject: Translated using Weblate (Uyghur)

Currently translated at 100.0% (99 of 99 strings)

Translation: openkylin-nile-new/audio
Translate-URL: http://weblate.openkylin.top/projects/openkylin-nile-new/audio/ug/
---
 audio/translations/ky.ts | 4 ++--
 audio/translations/ug.ts | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/audio/translations/ky.ts b/audio/translations/ky.ts
index 12078d3..280cc81 100644
--- a/audio/translations/ky.ts
+++ b/audio/translations/ky.ts
@@ -32,7 +32,7 @@
     <message>
         <location filename="../audio-device-control/ukmedia_device_control_widget.cpp" line="463"/>
         <source>Confirm</source>
-        <translation>بەكىتۉۉ</translation>
+        <translation>Ырастоосу</translation>
     </message>
 </context>
 <context>
@@ -45,7 +45,7 @@
     <message>
         <location filename="../audio-device-control/ukmedia_device_control_widget.cpp" line="149"/>
         <source>Confirm</source>
-        <translation>بەكىتۉۉ</translation>
+        <translation>Ырастоосу</translation>
     </message>
 </context>
 <context>
diff --git a/audio/translations/ug.ts b/audio/translations/ug.ts
index 8b404a3..ea66e15 100644
--- a/audio/translations/ug.ts
+++ b/audio/translations/ug.ts
@@ -12,7 +12,7 @@
     <message>
         <location filename="../audio.cpp" line="93"/>
         <source>UkccPlugin</source>
-        <translation></translation>
+        <translation>ukccplugin سىنىقى</translation>
         <extra-contents_path>/UkccPlugin/UkccPlugin</extra-contents_path>
     </message>
     <message>
@@ -398,7 +398,7 @@
     <message>
         <location filename="../ukmedia_output_widget.cpp" line="102"/>
         <source>Volume above 100% can cause sound distortion and damage your speakers.</source>
-        <translation>100%تىن يۇقىرى ھەجىملىك ئاۋازنىڭ بۇرمىلىنىشىنى كەلتۈرۈپ چىقىرىدۇ ۋە كانايىڭىزنى زەخىملەندۈرىدiۇ.</translation>
+        <translation>٪100تىن يۇقىرى ھەجىملىك ئاۋازنىڭ بۇرمىلىنىشىنى كەلتۈرۈپ چىقىرىدۇ ۋە كانايىڭىزنى زەخىملەندۈرىدۇ.</translation>
     </message>
     <message>
         <location filename="../ukmedia_output_widget.cpp" line="125"/>
